tools/chpl-language-server/src/chpl-language-server.py

```diff
@@ -976,13 +1063,10 @@ def _get_type_inlays(
         label = InlayHintLabelPart(str(type_))
         if isinstance(type_, chapel.CompositeType):
             typedecl = type_.decl()
-
-            if typedecl:
-                text = self.get_tooltip(typedecl, siblings, via)
-                content = MarkupContent(MarkupKind.Markdown, text)
-                label.tooltip = content
```

Comment:
How come you remove the tooltip thing?

Reply:
After doing significant research into the LSP spec, the tooltip here gets resolved for us if we leave it as `None`. So we can have less code for the same features